.github/actions/test-khiops-install/action.yml

```diff
@@ -9,14 +9,14 @@ runs:
     - name: Print status for debugging
       shell: bash
       # We force the process number to 4 because the runner has only 2 proc.
-      # With 2 proc khiops-env switches to serial and mpi is not used.
+      # With 2 proc khiops_env switches to serial and mpi is not used.
```

Comment:
In commit https://github.com/KhiopsML/khiops/commit/a98e684f40185467c99cc9b15c3cb1754e52490c's message, I would do the following updates:

- remove "The MODL binary is used to get proc number in khiops_env"
- replace "Linux" with "UNIXes", or "Linux and Mac OS" everywhere needed